client/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:10.15-alpine
+WORKDIR /usr/src/app
+COPY package.json yarn.lock ./
+RUN yarn install
+COPY . .
+EXPOSE 1729
+ENV NODE_ENV docker
+CMD ["yarn", "start"]
```

Comment:
I think the proper way of doing it is to do `yarn build` to produce a "production" build and then do `node build/index.js`.
However while trying that I had issues with env variables (because yarn start uses the `.env` file by default, which `node` does not do) and it's also harder to debug because the errors reference the "compiled" js files rather than the actual typescript files. You can leave it like that for now I guess.